modules/oe_whitelabel_search/src/Form/SearchForm.php

```diff
@@ -48,55 +48,30 @@ public function buildForm(array $form, FormStateInterface $form_state, array $co
     $form_state->set('oe_whitelabel_search_config', $config);
     $input_value = '';
 
+    $form['#region'] = $config['form']['region'];
     if (!empty($config['input']['name'])) {
       $input_value = $this->getRequest()->get($config['input']['name']);
     }
 
     $form['search_input'] = [
-      '#prefix' => '<div class="bcl-search-form__group w-100">',
-      '#suffix' => '</div>',
       '#type' => 'textfield',
-      '#title' => $config['input']['label'],
+      '#title' => $this->t('Search'),
       '#title_display' => 'invisible',
       '#size' => 20,
       '#margin_class' => 'mb-0',
-      '#attributes' => [
-        'placeholder' => $config['input']['placeholder'],
-        'class' => [
-          $config['input']['classes'],
-          'rounded-0',
-          'rounded-start',
-        ],
-      ],
+      '#form_id' => $this->getFormId(),
+      '#region' => $config['form']['region'],
       '#default_value' => $input_value,
       '#required' => TRUE,
     ];
 
     $form['submit'] = [
+      '#type' => 'submit',
       '#input' => TRUE,
       '#is_button' => TRUE,
       '#executes_submit_callback' => TRUE,
```

Comment:
```suggestion
      '#value' => $this->t('Search'),
```
a simple submit button doesn't need #input => TRUE etc